tutos/dev/manual/application.wiki

```diff
@@ -813,6 +814,28 @@ lines are drawn on both windows.
 
 <<concepts| Widgets with Ocsigen-widgets >>
 
+For the rest of the tutorial, we use the following import :
+
+<<code language="ocaml" class="server"|
+[%%shared
+  (* Modules opened in the shared-section are available in client-
+     and server-code *)
+  open Eliom_content.Html.D
+  open Eliom_content
+  open Lwt
+  open Js_of_ocaml
```

Comment:
This one is not opened in Graffiti either.
Trys to make the code exactly like in Graffiti project.
If you open Js_of_ocaml you don't need the following module redefinitions.